source/OptimaJet.DWKit.StarterApplication/Optimajet.DWKit.StarterApplication.csproj

```diff
@@ -1,45 +1,48 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
upgrades and whitespace changes... cause I guess csproj doesn't use `\n`? idk. now it does :-)

anywho, add ?w=1 to view the diff without whitespace changes.